objects/src/testing/account_code.rs

```diff
@@ -7,16 +7,16 @@ use crate::accounts::AccountCode;
 const MASTS: [&str; 12] = [
     "0xff06b90f849c4b262cbfbea67042c4ea017ea0e9c558848a951d44b23370bec5",
     "0x8ef0092134469a1330e3c468f57c7f085ce611645d09cc7516c786fefc71d794",
-    "0xed87fcbdaf52f9d544d6362c0177f7294d917dfdbf48ee31b3df7b28be6f4ea0",
-    "0xf26ab9001bbe615e3f39e6bf5c12dd32754b99d9476c6dd003881c3c52a6bfc1",
-    "0x3f4b2dadf8a08e963898bc172f3fb1f4cbd7c560cb1d5f88eec582d0f9819edb",
-    "0x31c806646881424c912cf5af6ad8002fe1f24cdc83e769525dd6c68df94a8f3b",
-    "0x1b4fd4fc6d711276c4b25dc9e9b8dff6feb8f3762562fcecf45b85c7e4ed2437",
-    "0x8197f4791b030c86bd4e3e4a241de840de47fa1ec338b8668ddb37052eee00a6",
-    "0x16d574d4d5a70f83c86843a62a19ca647d2110c11cfd5cb2fc93a149f3c9c2c5",
-    "0x70573a57c1b809e6feacf3bfeda3c6a22e4ef018979d2a0f9d466e54ed9c44a5",
-    "0x60cc189b72db5eae2de193245a5c979bc8c322a98d3980bff068e1c80c066306",
-    "0x875149b782d20eeda59477e260c173fbb477d1f6f755820d0bfb214ce0ebe78b",
+    "0xd68cf600d75f4c8f47fde53e97dff4dd7fbf3abd0fee92fd92d09787075b7ad8",
+    "0xdd55fc036505f12a47afa4cbe558f3e4eaee91e7fb4ad946042d2eb9c73a5ca4",
+    "0xd25e1522abb81d7e1343dc0722514c587e0b0730de6f58888ca5d56e4332eae2",
+    "0x8f6d91270bb15c57084af3f07db8209d6088f6841c6447c295fa3a300b81740e",
+    "0x03bb20b622523663c1d47d823f50b97ae4d01a12bcdab3eff76473a570f08240",
+    "0x53d0de6a04f92507d1ae4da76b1f215876cd6cd2062adce1ccc2ed74808d9157",
+    "0x9f1275829532e8f5c200dfb369747261c635def6f20ba7478935cf67db3f4bff",
+    "0xd544028d21f646d1793e51f834f9b50fa33c695cc3587a3511e88284f23bd0ad",
+    "0x111bad19bc6817c665ef2f07ed3d129c38da92bdd6959c7f5af1a2754674f9eb",
+    "0x28f19823ef55b7dc956436a3cb1176dace4c6cd68a8c165290aa2e19f9fd29b7",
 ];
```

Comment:
After we updated the assembler, hashes of the procedures now change their order every time their hash is changed, so it is pretty painful to get the hashes to the correct order. (Once I had to update these hashes, so I was looking through the MAST forest to locate the correct hashes among the blocks). @bobbinth Do we have an easy way to obtain the hashes and names of the compiled procedures? I know that we can do so for the `KernelLibrary` from the `ModuleInfo`, but it turns out to be a problem if we need to obtain this info for a plain module. Probably, for the testing purposes we could create some instrument to obtain this info from a `&str` source.

Reply:
After https://github.com/0xPolygonMiden/miden-base/pull/856 is merged, this array will go way (i.e., we won't need to make any updates here when procedure MASTs change).